problems/[23.08.24] 198. House Robber.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+
+const rob = function (nums) {
+  if (!nums.length) return 0;
+  if (nums.length === 1) return nums[0];
+
+  const result = [];
```

Comment:
배열의 필요한 요소만큼 미리 선언해두고 사용하시면 시간복잡도를 개선할 수 있을 것 같아요.
```suggestion
  const result = new Array(nums.length);
```